products/sle/templates

```diff
@@ -289,7 +289,7 @@
                       backend => "s390x",
                       name => "zVM-vswitch-l2",
                       settings => [
-                        { key => "S390_NETWORK_PARAMS", value => "HostIP=10.161.155.@S390_HOST@/20 Hostname=s390vsl@S390_HOST@.suse.de Gateway=10.161.159.254 Nameserver=10.160.0.1 Domain=suse.de PortNo=0 Layer2=1 Portname=VSWNL2 ReadChannel=0.0.0800 WriteChannel=0.0.0801 DataChannel=0.0.0802" },
+                        { key => "S390_NETWORK_PARAMS", value => "HostIP=10.161.155.@S390_HOST@/20 Hostname=s390vsl@S390_HOST@.suse.de Gateway=10.161.159.254 Nameserver=10.144.53.53 Domain=suse.de PortNo=0 Layer2=1 Portname=VSWNL2 ReadChannel=0.0.0800 WriteChannel=0.0.0801 DataChannel=0.0.0802" },
```

Comment:
is this template file still used?
cause those s390_NETWORK_PARAMS are also outdated after #18176

Reply:
No, it's not used directly but people might still refer to it as reference